src/api/SnazzyButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Button, ButtonProps } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+
+interface SnazzyButtonProps extends ButtonProps {
+  isSnazzy?: boolean;
+}
+const useStyles = createUseStyles({
+  snazzyButton: {
```

Comment:
Read what the task is asking you to do. 
You have not used the given styles.

Reply:
I what styles was given it just said create a component

Reply:
It has a link to the Styles (css) that you need to use.